php/libraries/Database.class.inc

```diff
@@ -20,11 +9,8 @@ define("DEBUG", false);
  * Throughout the class, all input arrays follow the same structure:
  * $array['column_name'] = 'value'.
  *
- * @category Main
- * @package  Loris
- * @author   Alex Zijdenbos <zijdenbos@example.com>
- * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
- * @link     https://www.github.com/aces/Loris-Trunk/
+ * @author  Alex Zijdenbos <zijdenbos@example.com>
```

Comment:
Is it possible to also not require the `@author` tag? That's also unnecessary (it's in git) and often wrong (people just copy the boilerplate from other files and don't update it)

Reply:
Done